runIntegrationTests

```diff
@@ -155,14 +155,15 @@ function runTestsInGradle {
 	echo "Starting Integration Tests (using Smack ${SMACK_VERSION})…"
 
 	DISABLED_INTEGRATION_TESTS=()
-	#DISABLED_INTEGRATION_TESTS+=(MoodIntegrationTest)
+	#Fails only in Java8, only on Travis
+	DISABLED_INTEGRATION_TESTS+=(MoodIntegrationTest)
 	#DISABLED_INTEGRATION_TESTS+=(MultiUserChatIntegrationTest)
```

Comment:
Do the tests that aren't disabled want deleting, or kept around as clues for future contributors for the time being?

Reply:
I really like what you currently have.